crates/data-transformer/src/transformer.rs

```diff
@@ -43,7 +43,8 @@ pub fn transform(
 }
 
 fn split_expressions(input: &str, db: &SimpleParserDatabase) -> Result<Vec<Expr>> {
-    let expr = parse_expression(input, db)?;
+    let input = format!("({input},)");
```

Comment:
Could you explain this?

Reply:
We have to pass the arguments as a valid cairo expression to the parser. I though that it'd be easiest to just wrap everything in tuple since we already had code to handle that - since previous version of calldata passing used `--calldata (1_u8, 234)` syntax.

Reply:
Maybe there should be a short comment? It seems unclear to me too.

Reply:
Added a comment